packages/genesys-spark-components/src/components/stable/gux-listbox/option-group/gux-option-group.tsx

```diff
@@ -9,24 +12,78 @@ import { Component, h, Host, JSX, Prop } from '@stencil/core';
   shadow: true
 })
 export class GuxOptionGroup {
+  private rootParent: HTMLElement;
+  private parentObserver: MutationObserver;
+
+  @Element() root: HTMLGuxOptionGroupBetaElement;
+
   @Prop()
   label!: string;
 
   // @Prop()
   // disabled: boolean;
   // TOOD: Add disabled state once design work is complete
 
+  @State()
+  filtered: boolean = false;
+
+  @State()
+  showDivider: boolean = true;
+
+  componentDidLoad(): void {
+    this.rootParent = getClosestElement(
+      'gux-dropdown',
```

Comment:
Should this be `gux-listbox`?

Reply:
Replace this with just getting the parentNode because the function getClosestElement seemed to be skipping the listbox wrapper